docs/ManualDeploymentGuideScalarDLOnEKS.md

```diff
@@ -1,74 +1,53 @@
 
-Note: Please see the **Subscribe to Scalar products from AWS Marketplace** section in the above document.  
+## Step 3. Set up a database for ScalarDL Ledger
 
-## Step 2. Set up a database for Scalar DL Ledger
-
-Scalar DL Ledger uses Scalar DB in its internal to access a database and Scalar DB supports [several databases](https://github.com/scalar-labs/scalardb/blob/master/docs/scalardb-supported-databases.md). You need to prepare a database before you deploy Scalar DL Ledger.  
-
-Please refer to the following document for more details.  
-
-* [Set up a database for Scalar DB/Scalar DL deployment on AWS](./SetupDatabaseForAWS.md)
-
-## Step 3. Create an EKS cluster
-
-Create an EKS cluster for the deployment of Scalar DL Ledger. Please refer to the following document for more details.  
-
-* [Create an EKS cluster for Scalar Products]() // TODO: Create a new document
+ScalarDL Ledger uses ScalarDB in its internal to access a database and ScalarDB supports [several databases](https://github.com/scalar-labs/scalardb/blob/master/docs/scalardb-supported-databases.md). You need to prepare a database before you deploy ScalarDL Ledger. See [Set up a database for ScalarDB/ScalarDL deployment on AWS](./SetupDatabaseForAWS.md) for more details.
 
 ## Step 4. Create a bastion server
 
-For executing some tools to deploy and manage Scalar DL Ledger on EKS, you need to prepare a bastion server in the same VPC of the EKS cluster you created in **Step 3**. Please refer to the following document for more details.  
-
-* [Create a bastion server]() // TODO: Create a new document
+For executing some tools to deploy and manage ScalarDL Ledger on EKS, you need to prepare a bastion server in the same VPC of the EKS cluster you created in **Step 2**. See [Create a bastion server](./CreateBastionServer.md) for more details.
 
 ## Step 5. Prepare a custom values file of Helm 
 
-You need to configure a custom values file for the Helm Chart of Scalar DL Ledger and Scalar DL Schema Loader based on your environment (e.g., access information of the database you created in **Step 2**). Please refer to the following document for more details.  
-
-* [Configure a custom values file of Scalar Helm Chart]() // TODO: Create a new document in the Scalar Helm Chart repository
-
-## Step 6. Deploy Scalar DL Ledger using Scalar Helm Chart
-
-Deploy Scalar DL Ledger on your EKS cluster using Scalar Helm Chart. Please refer to the following document for more details.  
+You need to configure a custom values file for the Helm Chart of ScalarDL Ledger and ScalarDL Schema Loader based on your environment (e.g., access information of the database you created in **Step 3**). See [Configure a custom values file for Scalar Helm Charts](https://github.com/scalar-labs/helm-charts/blob/main/docs/configure-custom-values-file.md) for more details.
```

Comment:
The following assumes we need to configure two custom values files (one for ScalarDL Ledger and one for ScalarDL Schema Loader [for Ledger]).

```suggestion
To perform a task, like accessing information in the database that you created in **Step 3**, you must configure custom values files for the Scalar Helm Charts for both ScalarDL Ledger and ScalarDL Schema Loader (for Ledger) based on your environment. For details, see [Configure a custom values file for Scalar Helm Charts](https://github.com/scalar-labs/helm-charts/blob/main/docs/configure-custom-values-file.md).
```

If only one custom values file needs to be configured, let's use the following:

```suggestion
To perform a task, like accessing information in the database that you created in **Step 3**, you must configure a custom values file for the Scalar Helm Chart for ScalarDL Ledger and ScalarDL Schema Loader (for Ledger) based on your environment. For details, see [Configure a custom values file for Scalar Helm Charts](https://github.com/scalar-labs/helm-charts/blob/main/docs/configure-custom-values-file.md).
```